src/app/views/sessions/signin/signin.component.ts

```diff
@@ -228,41 +229,9 @@ export class SigninComponent implements OnInit, OnDestroy, AfterViewInit {
                 console.error(err);
               },
             });
-            this.ws.call('failover.disabled.reasons').pipe(untilDestroyed(this)).subscribe({
-              next: (reasons) => {
-                this.checkingStatus = false;
-                this.haDisabledReasons = reasons;
-                this.showReasons = false;
-                if (reasons.length === 0) {
-                  this.haStatusText = this.translate.instant('HA is enabled.');
-                  this.haStatus = true;
-                } else if (reasons.length === 1) {
-                  if (reasons[0] === FailoverDisabledReason.NoSystemReady) {
-                    this.haStatusText = this.translate.instant('HA is reconnecting.');
-                  } else if (reasons[0] === FailoverDisabledReason.NoFailover) {
-                    this.haStatusText = this.translate.instant('HA is administratively disabled.');
-                  } else {
-                    this.haStatusText = this.reasonText[this.disabledReason(reasons[0])];
-                  }
-                  this.haStatus = false;
-                } else {
-                  this.haStatusText = this.translate.instant('HA is in a faulted state');
-                  this.showReasons = true;
-                  this.haStatus = false;
-                }
-                this.window.sessionStorage.setItem('ha_status', this.haStatus.toString());
-                if (this.canLogin()) {
-                  this.loginToken();
-                }
-              },
-              error: (err) => {
-                this.checkingStatus = false;
-                console.error(err);
-              },
-              complete: () => {
-                this.checkingStatus = false;
-              },
-            });
+
+            this.store$.dispatch(loadHaStatus());
```

Comment:
Only part of failover logic is handled here.
I think it may be easier to revert to old code here and keep store for logged-in logic only.
We can later move failover status here to a separate store.